tests/conftest.py

```diff
@@ -335,3 +335,34 @@ def VMEC_save(SOLOVEV, tmpdir_factory):
     )
     desc = Dataset(str(SOLOVEV["desc_nc_path"]), mode="r")
     return vmec, desc
+
+
+@pytest.fixture(scope="session")
+def SOLOVEV_Poincare(tmpdir_factory):
```

Comment:
what makes this use poincare? I thought we didn't support it from the input file?

Reply:
You are correct there was nothing that made it Poincare. It was unused, I guess Dario put it to use later just incase we support poincare from input files. I deleted it